core/threaded/scheduler_LET.c

```diff
@@ -342,7 +370,8 @@ void lf_sched_free() {
     free(_lf_sched_instance->_lf_sched_triggered_reactions);
     free(_lf_sched_instance->_lf_sched_executing_reactions);
     lf_semaphore_destroy(_lf_sched_instance->_lf_sched_semaphore);
-    free(worker_is_in_workforce);
+    free(_lf_sched_worker_is_in_workforce);
+    vector_free(&_lf_sched_transitioning_reactors);
```

Comment:
Shouldn't this free be guarded by `ifdef MODAL_REACTORS`?

Reply:
Yes, you are right. Thanks!